tests/features/taxonomy_browser.feature

```diff
@@ -0,0 +1,94 @@
+@api
+Feature: Taxonomy browser
+  In order to test ‘Taxonomy browser’ feature
+  As an administrator
+  I want to make sure ‘Taxonomy browser’ allows creating blocks with taxonomy vocabularies
+
+  Background:
+    Given I am logged in as a user with the 'administrator' role
+    When I visit "/admin/config/taxonomy_browser/settings_en"
+    When I check "edit-taxonomy-browser-vocabulary-2"
+    And  I press "Save configuration"
+    Then I should see the text "The configuration options have been saved."
+
+  @api
+  Scenario: Administrator user creates a new view to show the taxonomy block.
+    Given a content view with machine name "view_articles" is available
+    When I go to "/admin/structure/views/view/view_articles"
+    Then the response status code should be 200
+    Then I should see the text "Master details"
+    #Configure contextual
+    When I go to "/admin/structure/views/nojs/add-item/view_articles/default/argument_en"
+    Then I check "Content: Has taxonomy term ID (with depth)"
+    And I press the "Add and configure contextual filters" button
+    When I select the radio button "Provide default value"
+    And I fill in "Fixed value" with "fixed_value"
+    Then I check "Override title"
+    And I fill in "edit-options-title" with "%1"
+    And I press the "Apply" button
+    And the module is enabled
+      | modules         |
+      | easy_breadcrumb |
```

Comment:
I don't understand why this module is required to create a view.

Reply:
I was trying to make some kind of "user stories" from the documentation (https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?spaceKey=MULTISITE&title=Taxonomy+browser+-+documentation) . Removed.